### ISSUE TITLE
Learning Rate Matters: Vanilla LoRA May Suffice for LLM Fine-tuning

### ISSUE THREAD

**Issue by ddelalamo-takeda** [delalamo/PubMedFetcher#44]
https://arxiv.org/pdf/2602.04998
